types/application.go

```diff
@@ -20,3 +21,13 @@ type Application interface {
 	// Query for state
 	Query(query []byte) (code CodeType, result []byte, log string)
 }
+
+// Some applications can choose to implement ValidatorAware
+type ValidatorAware interface {
+
+	// Give app initial list of validators upon genesis
+	InitValidators([]*Validator)
```

Comment:
should we have a ret code here so apps can reject the validator set? maybe they have an expected validator set in their genesis (that hashes into their merkle root) and they want this to match?